frontend/apps/filemanager/filemanagershortcuts.lua

```diff
@@ -18,21 +18,21 @@ local FileManagerBookmark = Menu:extend{
     is_borderless = true,
 }
 
-function FileManagerBookmark:init()
+function FileManagerShortcuts:init()
     self.item_table = self:genItemTableFromRoot()
     Menu.init(self)
 end
 
-function FileManagerBookmark:genItemTableFromRoot()
+function FileManagerShortcuts:genItemTableFromRoot()
     local item_table = {}
-    local favorite_folder = G_reader_settings:readSetting("favorite_folders") or {}
+    local shortcut_folder = G_reader_settings:readSetting("shortcut_folders") or {}
```

Comment:
I would name (the like menu item text) the variable and setting `folder_shortcuts` here and everywhere.